packages/wallet-frontend/src/contexts/Application.jsx

```diff
@@ -35,6 +35,9 @@ export default function Provider({ children }) {
       if (type === 'walletUpdateInbox') {
         dispatch(updateInbox(JSON.parse(data)));
       }
+      if (type === 'walletAcceptOfferResponse') {
+        alert(data.outcome);
```

Comment:
I don't think we want to pop up and alert every time an offer was accepted?

Reply:
I think for tomorrow we do. Eventually this is supposed to be a toast.

Reply:
This is the only way the dapp encouragement instructions work